new-frontiers-14/frontier-station-14
# NFMS Medical POI. Name to be determined.

## About the PR



This PR adds a new POI. The intention is for this POI to serve as a foundation for getting the broader medical systems in place, tested and operating. I understand that there is an official POI coming in the future. I thought it would be helpful to have something as a stand in until that time comes. I also understand that there is a goal to have rotating POIs in the future, to which this may be suitable to contribute.

I believe I have taken this project as far as I can. Which makes me frustrated, but I also wish to be honest and practical. After a discussion with a few other people in the dev channel I am putting the POI as I have crafted it down on the table for it to be picked up and moved along. I understand that by doing so I am surrendering control over the project and do so willingly. But I hope it's a help. 

This POI is absolutely unfinished. There are many pieces that will still need to be completed that are above my skill level to do in a reasonable time frame. As such other community members with experience in these matters are invited to, and have expressed interest in taking this project the rest of the way. Cheers to Whatstone and Salvantrix!

What is still required:

1) New jobs and spawns.
- Director of Care DoC. The "DoC" is a whitelisted position that operates the station and controls ID/pay/so on. Calling them the DOC is funny and is non negotiable. 
- Medic. Unlimited number. Same hours as a pilot. No other restrictions. Don't try to lock it to like 3 medics a shift you weirdos.
2) New PDA for DoC and Medic. 
3) New ID for DoC and Medic. 
4) New security clearance levels and associated doors for the medical center. DoC level should be for the brig and DoC office. Medic for the rest of the facility. No reason to split up chemistry. Command level access for power/atmos as standard.
5) Addition of fuel vending machine and CiviMed vending machine to lobby. Yay my vending machine gets a chance to shine.
6) Medical ship spawning control console.
7) This station does not have shuttle service as there are medic locked gates and lack of control over landing points makes me uneasy. But ensuring that docks and EVA ports are accessible and serviceable. 
8) Addition of appropriate cargo summoning services when they are added in.
9) Balance passes that remove equipment that pertains to the creation of medical goods (I.E. microwave removal) must allow for alternative crafting recipes that are gate kept by those machines. So if a machine such as a microwave is removed a new recipe for advanced topicals must be provided. This POI should be capable of producing all contemporary medicines and procedures sans cloning. This will need to be maintained for all future upstream medical releases. So probably best to just let them keep a microwave.
10) The smaller docking bay on the medic side could possibly use some thought. It is currently locked off, I am not sure if there is a good way to label those docks so that people incoming know that is NOT a public dock. But I think they, or at least one, should remain.
11) Public restroom may allow for too simple an entry process and may need to be moved/destroyed.

Other issues and concerns may arise. This is largely a laundry list of stuff that is above my pay grade or beyond my skill. 

## Why / Balance


The community has expressed a desire for a new POI to manage medical vehicle spawning and provide basic medical infrastructure. My hope is that this addresses that need.

When establishing what equipment the POI would have I attempted to operate within the guidelines of what does a Caduceus have? Minus the cloning system. Conceptually the idea would be that such a ship was stripped down to serve as the material foundation for an expeditionary medical center build into an asteroid.

## How to test


When determining how critically to scrutinize this POI remember that I am a new player to SS14, a new player to Frontier and my current PRs consist of a ship that hasn't been accepted, a locker that broke space and time and a vending machine that I made green instead of blue. Treat this as highly suspect.

* Atmos, disposals, firelocks, power, all essential systems need a more seasoned eye than mine to double check.
* The stations maps are not up and running. Beacons are largely in place but I was unable to get that element operational.
* POI name, position and icon color likely could all use addressing.
* There are likely huge and embarrassing oversights in how the POI functions. I've never set one up before and I was working through monkey see monkey do.
*Doors are temporary stand ins.
*Ensure all computers and similar devices are appropriate to meet the minimum needs of a SR type position on the station in addition to ensuring the POI can function independently.
*Ensure "brig" area is accessible to DoC and functions appropriately.
*Triple check that I didn't fuck up the atmos or create an unintentional problem.

## Media


![ss+(2024-08-31+at+07 05 07)](https://github.com/user-attachments/assets/301f8dc7-78c3-492a-9232-8f78b2eea396)
![ss+(2024-08-31+at+07 06 56)](https://github.com/user-attachments/assets/5608d8da-50f8-4d9a-9f81-4412fe0882ec)
![ss+(2024-08-31+at+07 06 24)](https://github.com/user-attachments/assets/7c967d6a-a452-4295-8572-279a924a3c12)
![ss+(2024-08-31+at+07 07 28)](https://github.com/user-attachments/assets/8ccc0ddd-1099-44dd-b646-263e78b7cd8a)
![ss+(2024-08-31+at+07 07 54)](https://github.com/user-attachments/assets/5a84e1c8-5787-4a0b-aa0e-eb0f2241f66e)
![ss+(2024-08-31+at+07 08 40)](https://github.com/user-attachments/assets/1b0c256c-aeca-4c28-843c-206fecfca05e)

## Breaking changes


This does not replace or change anything that currently exists.

**Changelog**
